babel.config.js

```diff
@@ -21,5 +21,7 @@ module.exports = {
   plugins: [
     '@babel/plugin-proposal-class-properties',
     '@babel/plugin-proposal-object-rest-spread',
+    '@babel/plugin-proposal-nullish-coalescing-operator',
```

Comment:
Otherwise jest won't be able to handle TS 3.7 nullish and optional chaining later on. (babel-preset-typescript doesnt include that)